Playmate/View Controllers/Home Tab/MapPinsViewController.m

```diff
@@ -28,16 +35,25 @@ @implementation MapPinsViewController
 - (void)viewDidLoad {
     [super viewDidLoad];
     
+    self.appliedFilters = NO;
+    self.filters = nil;
+    
     self.mapView.delegate = self;
     self.mapView.showsUserLocation = YES;
     
+    self.currentAnnotations = [[NSMutableArray alloc] init];
+    
     isFirstTimeGettingLocation = YES;
     
     [CLLocationManager locationServicesEnabled];
     
     [self initPinLocationManager];
     
-    [self fetchData];
+    if (self.appliedFilters == YES) {
+        [self fetchDataWithFilters:self.filters];
+    } else {
+        [self fetchData];
+    }
```

Comment:
I feel [self fetchData] should internally check for `self.appliedFilters == YES`
Because `self.appliedFilters ` is a global property, and `[self fetchData] ` has access to it.